drivers/ipcc/ipcc_register.c

```diff
@@ -176,7 +176,7 @@ int ipcc_register(FAR struct ipcc_lower_s *ipcc)
 
   /* nxsem_init can't really fail us if we provide it with valid params */
 
-  nxsem_init(&priv->exclsem, 0, 1);
+  nxmutex_init(&priv->lock);
   nxsem_init(&priv->rxsem, 0, 0);
   nxsem_init(&priv->txsem, 0, 1);
```

Comment:
Let's disable priority inheritance here

Reply:
Since many place has the similar issue, let's do in the next patch: all nxsem_set_protocol will be removed, the default setting is good in all case.